packages/subgraph/test/validation/aggregateValidators.ts

```diff
@@ -96,6 +98,68 @@ export const validateATSEntity = (
     ).to.equal(expectedTotalAmountTransferredUntilUpdatedAt);
 };
 
+export const validateASTEntityAndItsLogEntry = (
+    graphATSData: IAccountTokenSnapshot
+) => {
+    const accountTokenSnapshotLogs: IAccountTokenSnapshotLog[] = graphATSData.accountTokenSnapshotLogs;
```

Comment:
this can maybe be squished into one line:
` const accountTokenSnapshotLog: IAccountTokenSnapshotLog = graphATSData.accountTokenSnapshotLogs[0];`